README.md

```diff
@@ -75,13 +75,36 @@ This is one of three components required for Project Lightspeed. Project Lightsp
 
 ## Getting Started
 
+## Setup
+
+### Docker
+
+1. Install [git](https://git-scm.com/downloads)
+1. Build the image from the master branch with:
+
+    ```sh
+    docker build -t grvydev/lightspeed-react https://github.com/GRVYDEV/Lightspeed-react.git
+    ```
+
+1. Run it with
+
+    ```sh
+    docker run -it --rm \
+      -p 8000:80/tcp -p 8080:8080/tcp \
+      -e WEBSOCKET_HOST=localhost -e WEBSOCKET_PORT=8080 grvydev/lightspeed-react
```

Comment:
If someone can check my ports setup is correct that would be nice (not too sure about other services using the websocket etc.)

Reply:
Granted I don't have my local setup working yet but I think `-p 8080:8080/tcp` needs to be removed. The container `lightspeed-webrtc` uses `8080` ( especially when using `network_mode: host`). So for `lightspeed-react` let's do `8000:80`.